### PR TITLE
BZ1589142 - Adding section for determining internal and external master URLs

### PR DESCRIPTION
**Pull request by kelbrown20**
For Version 3.11
Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1589142

Ready for QA: @zhaozhanqi 

Preview: https://deploy-preview-41417--osdocs.netlify.app/openshift-enterprise/latest/day_two_guide/environment_health_checks.html#connectivity-on-master-hosts

### CODE REVIEW

File: day_two_guide/topics/network_connectivity.adoc [hunk 1]
```diff
@@ -83,6 +83,20 @@ $ curl -k https://master.example.com:443/healthz
 ok
 ----
 
+*Determining internal and external master URLs*
+
+The following commands can be used to determine URLs used by the internal cluster and external clients. Example URLs are found in the previous example.
```
Comment (skrthomas) [inline day_two_guide/topics/network_connectivity.adoc:88]:
```suggestion
The following commands can be used to determine the URLs  that are used by the internal cluster and external clients. Example URLs are found in the previous example.
```

**Review by skrthomas — COMMENTED** [review]
One small wording suggestion. LGTM otherwise :)

File: day_two_guide/topics/network_connectivity.adoc [hunk 2]
```diff
@@ -83,6 +83,20 @@ $ curl -k https://master.example.com:443/healthz
 ok
 ----
 
+*Determining internal and external master URLs*
+
+The following commands can be used to determine the URLs that are used by the internal cluster and external clients. Example URLs are found in the previous example.
+
+To determine the Internal cluster URL:
```
Comment (bergerhoffer) [inline day_two_guide/topics/network_connectivity.adoc:90]: Does "Internal" and "External" (below) really have to be capitalized, or can they be lowered?

**Review by bergerhoffer — COMMENTED** [review]
One quick comment before merging!

**Pull request merged**